src/ui/ResponsiveComponents/ResponsiveTabs.ts

```diff
@@ -374,4 +379,32 @@ export class ResponsiveTabs implements IResponsiveComponent {
 
     return dropdownHeaderLabel;
   }
+
+  private tabIsSelected(tab: Dom | HTMLElement) {
+    return $$(tab as HTMLElement).hasClass('coveo-selected');
+  }
+
+  private tabIsInDropdown(tab: Dom | HTMLElement) {
+    return $$(tab as HTMLElement).hasClass(ResponsiveTabs.TAB_IN_DROPDOWN_CSS_CLASS);
+  }
+
+  private get tabsInTabSection(): HTMLElement[] {
+    let tabsInSection = [];
+    each(this.tabSection.el.children, childElement => {
+      if (Utils.isHtmlElement(childElement)) {
+        let child = $$(<HTMLElement>childElement);
+        if (!this.tabIsInDropdown(child) && child.hasClass(Component.computeCssClassNameForType(this.ID))) {
```

Comment:
I think it would be easier to understand this condition if `child.hasClass(Component.computeCssClassNameForType(this.ID)` was extracted into a local variable.